src/pages/dashboard/dashboard.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+import { NavController, NavParams } from 'ionic-angular';
+import { JournalPage } from '../journal/journal';
+import { LdtmModePage } from '../ldtm-mode/ldtm-mode';
+import { PolicyDataPage } from '../policy-data/policy-data';
+import { ManageDeviceDebriefStoragePage } from '../manage-device-debrief-storage/manage-device-debrief-storage';
+import { TrainerModePage } from '../trainer-mode/trainer-mode';
+import { DashboardComponentLinks } from './enums/dashboard-component-links';
+
+@Component({
+  selector: 'page-dashboard',
+  templateUrl: 'dashboard.html',
+})
+export class DashboardPage {
+
+  title: string = 'Dashboard';
+
+  constructor(public navCtrl: NavController, public navParams: NavParams) {
+    navCtrl.remove(0);
+  }
+
+  ionViewDidLoad() {
+    console.log('ionViewDidLoad DashboardPage');
+  }
+
+  goToPage(page: string) {
+    if (page === DashboardComponentLinks.JournalPage) return this.navCtrl.push(JournalPage);
```

Comment:
Looks much better, I would change these if statement to switch and then it would be mind-blowing piece of code ;)